Ginger/GingerCoreNET/Run/RunsetExecutor.cs

```diff
@@ -468,7 +483,10 @@ public async Task RunRunset(bool doContinueRun = false)
                     RunSetConfig.LastRunsetLoggerFolder = "-1";   // !!!!!!!!!!!!!!!!!!
                     Reporter.ToLog(eLogLevel.INFO, string.Format("Reseting {0} elements", GingerDicser.GetTermResValue(eTermResKey.RunSet)));
                     mStopwatch.Reset();
-                    ResetRunnersExecutionDetails();
+                    if (WorkSpace.Instance.RunsetExecutor.RunSetConfig.ReRunConfigurations.Active && WorkSpace.Instance.RunsetExecutor.RunSetConfig.ReRunConfigurations.ReferenceExecutionID != null)
+                    {
+                        ResetRunnersExecutionDetails();
+                    }                   
```

Comment:
Add else part to have proper error message that Reference Execution id is empty or invalid